diagrams.md

```diff
@@ -0,0 +1,548 @@
+        Choice property_accountable
+        Choice gfe_gfp_furnished
+        String dpas_unit_identification_code
+    }
+    PERIOD {
+        GUID sys_id PK
+        Choice period_type "BASE, OPTION"
+        Integer period_unit_count
+        Choice period_unit "DAY, WEEK, MONTH, YEAR"
+        Integer option_order
+    }
+    REQUIREMENTS-COST-ESTIMATE {
+        GUID sys_id PK
+        String surge_capabilities
+        Integer contracting_office_fee_pct
+    }
+    SERVICE-OFFERING {
+        GUID sys_id PK
+        String description
+        Choice service_offering_group
+        Integer sequence
+        String name
+    }
+    SELECTED-SERVICE-OFFERING {
+        GUID sys_id PK
+        List classification_instances FK
+        List estimated_environment_instances FK
+        Reference service_offering FK
+        String other_service_offering
+    }
+    ESTIMATED-ENVIRONMENT-INSTANCE {
+        Extends ENVIRONMENT-INSTANCE "inherits cols"
+        GUID sys_id PK
+        String dow_task_number
+        Currency monthly_price
+    }
+    %% CURRENT-ENVIRONMENT-INSTANCE {
+    %%     Extends ENVIRONMENT-INSTANCE "inherits cols"
+    %% }
+    %% COMPUTE-ENVIRONMENT-INSTANCE {
+    %%     Extends ENVIRONMENT-INSTANCE "inherits cols"
+    %% }
+    %% DATABASE-ENVIRONMENT-INSTANCE {
+    %%     Extends ENVIRONMENT-INSTANCE "inherits cols"
+    %% }
+
+    CLASSIFICATION-INSTANCE {
+        GUID sys_id PK
+        Reference classification_level FK
+        List selected_periods FK
+        String dow_task_number
+        Currency monthly_price
+        Choice need_for_entire_task_order_duration
+        String usage_description
+    }
+    CLASSIFICATION-LEVEL {
+        GUID sys_id PK
+        Choice classification "U, S, TS"
+        Choice impact_level "IL2, IL4, IL5, IL6"
+        String display "calculated value"
+    }
+    ENVIRONMENT-INSTANCE {
+        GUID sys_id PK
+        Reference classification_level FK
+        List selected_periods FK
+        Choice need_for_entire_task_order_duration "true=select all periods"
+        String instance_name
+        Choice instance_location "CSP, ON_PREMISE, HYBRID"
+        Choice csp_region "empty; TODO change to ref Region table"
+        String operating_system_licensing
+        Choice pricing_model "RESERVED, PAY_AS_YOU_GO"
+        Date pricing_model_expiration
+        Choice performance_tier "empty; TODO"
+        Integer number_of_vcpus
+        Integer memory_amount
+        Choice memory_unit "GB, TB"
+        Choice storage_type "empty; TODO"
+        Integer storage_amount
+        Choice storage_unit "GB, TB; TODO add PB"
+        Integer data_egress_monthly_amount
+        Choice data_egress_monthly_unit "GB, TB; TODO add PB"
+    }
+    CURRENT-ENVIRONMENT {
+        GUID sys_id PK
+        String additional_information
+        Choice has_migration_documentation
+        Choice has_system_documentation
+    }
```

Comment:
I know the `Current Environment` table is in progress and being updated, but wondering if the below columns where intentionally left off
- `Boolean current_environment_exists`
- `List environment_instances` (might be changing/updating??)

Reply:
Yes, changes coming, but those two cols should be included.

Reply:
Updated in https://github.com/dod-ccpo/atat-snow/pull/193/commits/c78b5353e220ca508b434afedd69f6f1cfab306a